src/components/Instructions.component.js

```diff
@@ -1,21 +1,35 @@
 import React, { Component } from 'react'
 // TODO: uncomment this for Part 1
-// import PropTypes from 'prop-types'
+import PropTypes from 'prop-types'
 import './../styles/instructions.css'
 
 class Instructions extends Component {
   render() {
     return (
       <div className="instructions">
         Follow the instructions on the README to get started!
-        <img
-          className="instructions__logo"
-          src="https://uiuc.hack4impact.org/img/colored-logo.png"
-          alt="h4i logo"
-        />
+        {this.props.shouldDisplayName && (
+          <img
+            className="instructions__logo"
+            src="https://uiuc.hack4impact.org/img/colored-logo.png"
+            alt="h4i logo"
+          />
+        )}
+        {this.props.items.map(
+          (item, index) =>
+            index % 2 === 1 ? (
+              <div>{item.string.toUpperCase()}</div>
+            ) : (
+              <div>{item.string}</div>
+            )
```

Comment:
Good job using a ternary operator here. You could further simplify this by having the ternary operator only determine the string inside the `div` tags rather than duplicating the `div` tags.

```jsx
(item, index) =>
  <div>{index % 2 === 1 ? item.string.toUpperCase() : item.string}</div>
```